content/3d-main-window/Surfaces.md

```diff
@@ -17,29 +17,60 @@ in the **3D Main Window**.
 ![]({{< relref "" >}}images/3d-main-window/SurfacesImport.png)
 
 Imported surfaces are listed under the {{< image-in-text src="images/3d-main-window/SurfacesProjectTreeNode.png" >}} entry 
-in **Project Tree** allowing for change of name, file, and color.
+in the **Project Tree**. You can click on a surface to change i.e. the name or color of the surface, or change the file to read the surface data from. 
 
 ![]({{< relref "" >}}images/3d-main-window/SurfacesPropertyEditor.png)
 
+You can also change the depth of the surface by adding a depth offset (positive or negative). This could be used i.e. to import the same surface file multiple times and assign each instance of the surface a separate depth offset. When a depth offset is set, the offset value will be shown as part of the surface name in the project tree.
+
+![]({{< relref "" >}}images/3d-main-window/surface_multiple.png)
+
+{{% notice note %}}
+Multiple surfaces can be imported at the same time by choosing more than one input file in the file selection window that shows up when you start an import.
+{{% /notice %}}
+
+
 ## Surface Results
 
 Per default, ResInsight maps current **Cell Result** on a given surface. 
 If another result is to be displayed, specify the result under **Separate Surface Results** as shown below.
 
 ![]({{< relref "" >}}images/3d-main-window/SurfacesSeparateSurfaceResults.png)
 
-Clicking a surface of a **View** in **Project Tree** activates the Property Editor for allowing view settings as shown below. 
-Please note the **Depth Offset** capability to offset a surface in z-direction and initiate result mapping to the surface 
-at that particular depth.
+Clicking a surface of a **View** in the **Project Tree** activates the Property Editor for allowing view settings as shown below. 
+
 To change result displayed on the surface, please specify the desired result in the **Separate Result Reference**
 section of the Property Editor.
 
 ![]({{< relref "" >}}images/3d-main-window/SurfacesViewPropertyEditor.png)
 
 
-## Supported Surface Import Formats
+## Reloading Surfaces
+
+If you have modified a surface file using an external program, you can easily load the changes into ResInsight by using the reload surface feature. Bring up the context menu for the surface you want to reload (usually a mouse right-click) and choose  **Reload**. The views will automatically update with the new data (could take a few seconds).
```

Comment:
Use "right-click menu" instead of "context menu" Many users are more familiar with "right-click"